paychmgr/manager.go

```diff
@@ -309,14 +330,6 @@ func (pm *Manager) ListVouchers(ctx context.Context, ch address.Address) ([]*Vou
 	return ca.listVouchers(ctx, ch)
 }
 
-func (pm *Manager) NextNonceForLane(ctx context.Context, ch address.Address, lane uint64) (uint64, error) {
```

Comment:
NextNonceForLane is no longer needed because CreateVoucher is now inside the Manager